packages/gcp/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.6.2"
```

Since this is an enhancement rather than a bugfix.
```suggestion
- version: "1.7.0"
```